webui/react/src/components/MultiSelect.tsx

```diff
@@ -0,0 +1,85 @@
+import { Select } from 'antd';
+import { SelectValue } from 'antd/es/select';
+import React, { useCallback, useMemo } from 'react';
+
+import SelectFilter from './SelectFilter';
+
+interface LabeledValue {
+  label: number|string;
+  value: number|string;
+}
+
+interface Props {
```

Comment:
**chore:** allow for all the `Select` props to be accepted as well (look at `SelectFilter.ts`). So style, className, etc will all get accepted:
```
import { Select } from 'antd';

interface Props<T = SelectValue> extends SelectProps<T> { ... }

return (
  ...
  <Select {...props}>...</Select>
);
```